docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -11,15 +11,18 @@ UniquePersonList -right-> "*" Person
 UniqueDeliveryList -down-> "*" Delivery
 
 Person -up-> "*" Tag
-Person *--> Name
-Person *--> Phone
-Person *--> Email
+Person *--> "1" Name
+Person *--> "1" Phone
+Person *--> "1" Email
+Person *--> "1" Company
+Person *--> "*" Product
+Person -up-> "1" SupplierStatus
 
 Delivery *--> "*" Product
 Delivery *--> "1" DateTime
 Delivery *--> "1" Person
 Delivery *--> "1" SupplierIndex
 Delivery *--> "1" Quantity
 Delivery *--> "1" Cost
-Delivery *--> "1" Status
+Delivery *--> "1" DeliveryStatus
```

Comment:
Currently the class is Status, but I think this change should be implemented to DeliveryStatus to increase the overall code quality